src/util/kaldi-table-inl.h

```diff
@@ -1549,6 +1624,8 @@ class RandomAccessTableReaderScriptImpl:
     last_found_ = 0;
     script_.clear();
     current_key_ = "";
+    range_ = "";
+    data_rxfilename_last_ = "";
```

Comment:
it's not initialization. ok so I'll switch to clear(). The above line current_key_ = ""; was there before..